### PR TITLE
Update babel-preset-es2015 to babel-preset-env

### PR DESCRIPTION
**Pull request by leandrooriente**
# Why

- https://codeburst.io/babel-preset-env-cbc0bbf06b8f
- Babel warnings in all our projects.
- Smaller bundles than our actual solution.

## Acceptances (PRODUCTION API)
- https://babel-preset-env-acceptance.cheaptickets.nl
- https://babel-preset-env-acceptance.flugladen.at
- https://babel-preset-env-acceptance.budgetair.nl
- https://babel-preset-env-acceptance.vayama.com
- https://babel-preset-env-acceptance.vliegwinkel.nl

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-const babelPresetEs2015 = require('babel-preset-es2015');
+const babelPresetEnv = require('babel-preset-env');
 const babelPresetStage3 = require('babel-preset-stage-3');
 const babelPresetReact = require('babel-preset-react');
 const babelPluginTransformExportDefault = require('babel-plugin-transform-export-default');
 const babelPluginTransformEs2015ObjectSuper = require('babel-plugin-transform-es2015-object-super');
 const babelPluginTransformFunctionBind = require('babel-plugin-transform-function-bind');
 
+const targets = {
+  node: 'current',
```
Comment (yurist38) [inline index.js]: Should we maybe have a fixed Node version here? AFAIK we use v8 now everywhere.

File: index.js [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+  android: 30,
+  chrome: 50,
+  edge: 14,
```
Comment (yurist38) [inline index.js]: Why 14? We support starting from 15-th.

Comment (leandrooriente) [reply index.js]: Yeap, I don't have the list of supported browsers, so we need to update it.

**Comment by fahad19** [comment]
> I don't know exactly which browsers we support, so we should edit our supported browsers with the correct data.

https://github.com/Travix-International/postcss-travix#autoprefixing

**Comment by leandrooriente** [comment]
Hey @fahad19, @yurist38 sent to me the spreadsheet with official data. Our current supported browsers are:

```js
  chrome: 53,
  edge: 15,
  explorer: 11,
  firefox: 52,
  safari: 9,
```

**Comment by fahad19** [comment]
@leandrooriente, @yurist38: in that case, please update `postcss-travix` too accordingly (if needed). thanks!

**Comment by yurist38** [comment]
Did you guys test it somehow with our project(s)?

**Comment by leandrooriente** [comment]
@yurist38 I tested RWD changing package.json from `"babel-preset-travix": "~1.1.0",` to `file:../babel-preset-travix`

But I'll create acceptances today using the PR  babel-preset-travix instead of the npm version.

File: index.js [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
 
+const targets = {
+  node: 8,
+  chrome: 53,
+  edge: 15,
+  explorer: 11,
+  firefox: 52,
+  safari: 9,
+  ucandroid: 1
+};
```
Comment (rbardini) [inline index.js]: Can't we use queries like `last X versions` for well-behaved browsers? How often are we going to update these targets?

Comment (yurist38) [reply index.js]: I think it makes sense to follow the support table and be synchronized with it. It doesn't change much actually, so fixed versions look fine to me.

**Comment by leandrooriente** [comment]
Acceptances (PRODUCTION API)
- https://babel-preset-env-acceptance.cheaptickets.nl
- https://babel-preset-env-acceptance.flugladen.at
- https://babel-preset-env-acceptance.budgetair.nl
- https://babel-preset-env-acceptance.vayama.com
- https://babel-preset-env-acceptance.vliegwinkel.nl

**Comment by rbardini** [comment]
Do we have metrics with and without the new preset? Would be nice to know how much this change affects bundle sizes—even though we should migrate anyway.

**Comment by leandrooriente** [comment]
@rbardini Exactly the same size.
Probably because of the browsers that we support now.

**Comment by yurist38** [comment]
I've checked the whole flow on FL.AT (IE11), goes well. 👍

**Comment by rbardini** [comment]
Did some testing on CT.NL using IE11, seems to be working fine 🙂

**Comment by leandrooriente** [comment]
@fahad19 should we merge and publish a new Major?

File: index.js (hunk 3 above)
Comment (EduardTrutsyk) [reply index.js]: maybe better to use:
'last 2 versions',
'iOS >= 8',
'Safari >= 9',
'IE 11',
What do you think? and when we are going to merge this PR?
@fahad19 @yurist38 @rbardini

**Comment by EduardTrutsyk** [comment]
Redeployed ACCP with latest master branch:
* https://babel-preset-env-acceptance.cheaptickets.nl
* https://babel-preset-env-acceptance.flugladen.at
* https://babel-preset-env-acceptance.budgetair.nl
* https://babel-preset-env-acceptance.vayama.com
* https://babel-preset-env-acceptance.vliegwinkel.nl

@froskie @leandrooriente @yurist38 @rbardini @fahad19 @iwwwi 
Please review and let's merge it ;)

**Comment by leandrooriente** [comment]
As discussed with @fahad19 I'll release a new minor version of babel-preset-travix

**Pull request merged**